### PR TITLE
feat(experimental): `Tooltip` add new component

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Closes #

### CODE REVIEW

File: projects/demo/src/modules/experimental/tooltip/tooltip.module.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        TuiLinkModule,
+        TuiLoaderModule,
+        TuiInputModule,
+        PolymorpheusModule,
+        TuiModeModule,
+        TuiAddonDocModule,
+        CommonModule,
+        FormsModule,
+        TuiTextfieldControllerModule,
+        RouterModule.forChild(tuiGenerateRoutes(ExampleTuiTooltipComponent)),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/tooltip/tooltip.module.ts:36]:
```suggestion
        tuiGetDocModules(ExampleTuiTooltipComponent),
```

\+ drop  `TuiAddonDocModule`

File: projects/demo/src/modules/experimental/tooltip/tooltip.component.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    readonly example2: TuiDocExample = {
+        TypeScript: import('./examples/2/index.ts?raw'),
+        HTML: import('./examples/2/index.html?raw'),
+        LESS: import('./examples/2/index.less?raw'),
+    };
+
+    readonly example3: TuiDocExample = {
+        TypeScript: import('./examples/2/index.ts?raw'),
+        HTML: import('./examples/2/index.html?raw'),
+        LESS: import('./examples/2/index.less?raw'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/tooltip/tooltip.component.ts:27]:
```suggestion
    readonly example3: TuiDocExample = {
        TypeScript: import('./examples/3/index.ts?raw'),
        HTML: import('./examples/3/index.html?raw'),
        LESS: import('./examples/3/index.less?raw'),
```

File: projects/demo/src/modules/experimental/tooltip/examples/2/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+></tui-tooltip>
+
+<tui-input
+    tuiTextfieldSize="m"
+    class="input"
+    [tuiTextfieldLabelOutside]="true"
+    [(ngModel)]="value"
+></tui-input>
+<tui-tooltip
+    content="Set icon color with 'color'"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/experimental/tooltip/examples/2/index.html:21]: I've not noticed any color's change 🤔 
Screenshot 2023-11-20 at 16 06 04

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/experimental/tooltip/examples/import/import-module.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import {TuiHintModule} from '@taiga-ui/core';
+import {TuiTooltipModule} from '@taiga-ui/experimental';
+
+// ...
+
+@NgModule({
+  imports: [
+    // ...
+    TuiTooltipModule,
+    TuiHintModule,
```
Comment (waterplea) [inline projects/demo/src/modules/experimental/tooltip/examples/import/import-module.md:11]: We don't really need it, right?

File: projects/experimental/components/tooltip/tooltip.template.html [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<span
+    #driver="tuiHintHover"
+    automation-id="tui-tooltip__icon"
+    tuiAppearance="icon"
+    class="t-tooltip-icon"
+    [attr.tabindex]="describeId ? -1 : 0"
+    [tuiAppearanceState]="!!(driver | async) ? 'active' : null"
```
Comment (waterplea) [inline projects/experimental/components/tooltip/tooltip.template.html:7]: Unnecessary !!

File: projects/experimental/components/tooltip/tooltip.template.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<span
```
Comment (waterplea) [inline projects/experimental/components/tooltip/tooltip.template.html:1]: Why do we need this span and not just the `tui-icon`?

File: projects/demo/src/modules/experimental/tooltip/examples/2/index.html (hunk 3 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/experimental/tooltip/examples/2/index.html]: rewritten to appearance

File: projects/experimental/components/tooltip/tooltip.template.html (hunk 6 above)
Comment (vladimirpotekhin) [reply projects/experimental/components/tooltip/tooltip.template.html]: made with `tuiIconButton`, but without `polymorpheusOutlet`

**Review by splincode — APPROVED** [review]

**Review by waterplea — APPROVED** [review]
Looks like this is going to be a hostDirective in 4.0 )

**Pull request merged**